Remove admin show page in Participatory Process Groups

andreslucena opened the pull request:
#### :tophat: What? Why?

After saving or editing a Participatory Process Group I was redirected to a screen that shows information. That's weird as usually in other admin forms we just redirect to this same form. 

This PR fixes that and makes it consistent with the rest of the platform. 

#### :pushpin: Related Issues
 
- Related to https://github.com/decidim/decidim/issues/8100#issuecomment-886446468

#### Testing

1. Sign in as admin
2. Edit a participatory process group 
3. You're redirected to this same form

### :camera: Screenshots

![Selecció_1488](https://user-images.githubusercontent.com/717367/132495728-0460bca7-be98-49b5-9d5d-d7290f20563a.png)


:hearts: Thank you!

leio10 commented:
@andreslucena I've rebased this PR to develop and there are 2 tests failing related to participatory processes, can you check it?

andreslucena commented:
> @andreslucena I've rebased this PR to develop and there are 2 tests failing related to participatory processes, can you check it?

@leio10 I've checked it and fixed the tests. Now it's passing the CI.
